README.md

```diff
@@ -1,3 +1,12 @@
 # Rethnet
 
-This repository is only used for issues at the moment. Please take a look at the [Rethnet GitHub Project](https://github.com/orgs/NomicFoundation/projects/3) and [Hardhat repository](https://github.com/NomicFoundation/hardhat) instead.
+**Rethnet** is an Ethereum development runtime library - written in Rust - with bindings for the Node API (Typescript).
+Its goal is to provide a performant API for developers tooling, such as an EVM debugger or state inspector.
+
+Rethnet finds its origins in Hardhat, but is a complete rewrite of our Hardhat Node Typescript code to Rust, incorporating all of the lessons we have learned over the years, and much more to come.
+As such, it exists within - an experimental branch of - the [Hardhat monorepo](https://github.com/NomicFoundation/hardhat/tree/rethnet/main/).
```

Comment:
I'd tweak it in this direction:

``Rethnet finds its origins in Hardhat but will be a complete rewrite of our Hardhat Node Typescript code to Rust, incorporating all of the lessons we have learned over the years, and much more to come.
Right now it exists within - an experimental branch of - the [Hardhat monorepo](https://github.com/NomicFoundation/hardhat/tree/rethnet/main/), since we're starting by incrementally embedding specific runtime components into Hardhat's TypeScript runtime.``

Reply:
Incorporated your suggestions, but rephrased the "since we're starting..." as I felt your sentence can be interpreted to mean that the monorepo is merely a temporary solution (not the experimental branch)